RealEstate/apps/core/views.py

```diff
@@ -121,10 +136,65 @@ def _homebuyer_post(self, request, homebuyer, *args, **kwargs):
         return render(request, self.homebuyer_template_name, context)
 
     def _realtor_get(self, request, realtor, *args, **kwargs):
-        return render(request, self.realtor_template_name, {})
+        couple = Couple.objects.filter(homebuyer__user=request.user)
+        couples = Couple.objects.filter(realtor=realtor)
+        pendingCouples = PendingCouple.objects.filter(realtor=realtor)
+        house = House.objects.filter(couple=couple)
+        # Couple data is a list of touples [(couple1, homebuyers, isPending),
+        # (couple2, homebuyers, isPending)] There may be a better way to get
+        # homebuyers straight from couples, but I didn't see it in the model.
+        coupleData = []
+        isPending = True
+        hasPending = True if (len(pendingCouples) > 0) else False
+        for couple in couples:
+            homebuyer = Homebuyer.objects.filter(couple=couple)
+            coupleData.append((couple, homebuyer, not isPending))
+        for pendingCouple in pendingCouples:
+            pendingHomebuyer = PendingHomebuyer.objects.filter(
+                pending_couple=pendingCouple)
+            coupleData.append((pendingCouple, pendingHomebuyer, isPending))
+        context = {
+            'couples': coupleData,
+            'house': house,
+            'realtor': realtor,
+            'form': InviteHomebuyerForm(),
+            'hasPending': hasPending
+        }
+        return render(request, self.realtor_template_name, context)
 
     def _realtor_post(self, request, realtor, *args, **kwargs):
```

Comment:
Make all the same changes in the post method as described above in the get method.